scripts/resource.py

```diff
@@ -57,11 +57,22 @@ def cleaner_k8s(f):
     return f.lower()
 
 
+def cleaner_oci(f):
+    f = f.replace("-", "_")
```

Comment:
diagrams uses hyphen-case for image resource names. Please keep the consistency :)